app/e2e/client/rocketchat.e2e.room.js

```diff
@@ -97,7 +98,7 @@ export class E2ERoom {
 	}
 
 	isSupportedRoomType(type) {
-		return ['d', 'p'].includes(type);
+		return roomTypes.getConfig(type).allowRoomSettingChange({}, RoomSettingsEnum.E2E);
```

Comment:
Shouldn't be `allowMemberAction`?

Reply:
nop,   look: allow**RoomSetting**Change **RoomSettings**Enum

Reply:
This suggestion is only valid if my other suggestion to change the enum name is accepted.